### PR TITLE
Added the feature of showcasing recipie cards.

### PR DESCRIPTION
**Pull request by ABDULALI3468**
In this branch I did the following functionalities: 
1) Created the cards dynamically by fetching data from MEAL DB API
2) Styles the cards elegantly.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -4,9 +4,46 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
```
Comment (sntakirutimana72) [inline docs/index.html:7]: Kindly, consider removing this line as it already exists in the _**src/assets/styles.css**_ file.

File: src/index.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
-import onready from './app.js';
+// import onready from "./app.js";
 import './assets/styles.css';
 import './assets/popup.css';
+import displayCards from './displayCards.js';
 
-window.addEventListener('DOMContentLoaded', onready);
+window.addEventListener('DOMContentLoaded', () => {
+  displayCards();
+  // onready();
```
Comment (sntakirutimana72) [inline src/index.js:8]: Kindly consider removing those commented lines for now and you may import them later when it's time to use them.

**Review by sntakirutimana72 — CHANGES_REQUESTED** [review]
Hi @ABDULALI3468 

Your project looks great, however, there are a couple of things to change.

## REQUIRED CHANGES

Kindly refer to the below comment for required changes for more details

- [ ] Remove commented lines of code
- [ ] Remove duplicated style link references.

Cheers!! 🥂 

Cheers

**Comment by ABDULALI3468** [comment]
.

**Pull request closed without merging**